macros/database_specific_helpers/get_relation.sql

```diff
@@ -0,0 +1,14 @@
+{% macro get_relation(get_relation_name) %}
```

Comment:
```suggestion
{% macro get_relation(relation_name) %}
```

Thinking this makes it more clearer as it's not getting the relation name. It's the relation that is being looked up in the graph.

Reply:
Completely agree - I copied and pasted it and didn't even look, but 100% with you.